calicoctl/commands/node/diags.go

```diff
@@ -134,6 +136,9 @@ func runDiags(logDir string) {
 		fmt.Printf("No logs found in %s; skipping log copying", logDir)
 	}
 
+	// Try to copy logs from containers for hosted installs.
+	getDockerLogs(tmpLogDir)
```

Comment:
I think we should call this `getContainerLogs`, because in future if we want to add rkt logs or any other container runtime logs it sounds more generic